src/main/java/sirius/biz/storage/util/StorageUtils.java

```diff
@@ -90,9 +90,10 @@ public Collection<Extension> getStorageSpaces(ConfigScope scope) {
      *
      * @param key  the key to verify
      * @param hash the hash to verify
+     * @param validityDays the number of days the hash should be valid into the past
      * @return <tt>true</tt> if the hash verifies the given object key, <tt>false</tt> otherwise
      */
-    public boolean verifyHash(String key, String hash) {
+    public boolean verifyHash(String key, String hash, int validityDays) {
```

Comment:
By changing the signature of a public method, how is that not breaking?

Reply:
Simple, we overloaded the method, the old method is still there